pkg/types/aws/validation/platform.go

```diff
@@ -21,6 +21,10 @@ var kubernetesNamespaceRegex = regexp.MustCompile(`^([^/]*\.)?kubernetes.io/`)
 // openshiftNamespaceRegex is used to check that a tag key is not in the openshift.io namespace.
 var openshiftNamespaceRegex = regexp.MustCompile(`^([^/]*\.)?openshift.io/`)
 
+// userTagLimit is defined in openshift/api
+// https://github.com/openshift/api/blob/1265e99256880f8679d1b74561c0bc7932067c43/config/v1/types_infrastructure.go#L370-L376
+const userTagLimit = 25
```

Comment:
Installer also creates tags so I'm not sure if 25 is the limit and I think it is a little lower than that.

Reply:
The limit is 50, with 25 reserved for openshift and 25 for users.

Reply:
If https://github.com/openshift/api/pull/1272 merges, we will adjust this number too

Reply:
Sounds good to me, thanks for the updates @patrickdillon